### ISSUE TITLE
[2.5.x] Build Overview Documention outdated

### ISSUE THREAD

**Issue by wjtan** [playframework/playframework#6030]
The list of useful import for build.scala is outdated:

```
import sbt._
import Keys._
import play.Play.autoImport._
import PlayKeys._
```

**Comment by wsargent** [comment]
@wjtan Please provide more information.  What's outdated?  What error message are you getting?  Which build.scala file are you talking about?  Are you suggesting the given imports as being "useful" or as outdated and due to be removed?

**Comment by wjtan** [comment]
For a play project, you can either use "build.sbt", or define a "Build.scala". For Build.scala, you have to explicitly import the sbt packages for play. But the sbt packages for play has been moved. Therefore the given imports in the documentation(https://www.playframework.com/documentation/2.5.x/BuildOverview#using-scala-for-building) is outdated and should be updated to the new packages.

The correct import should be:
```
import sbt._
import Keys._
import play.sbt._
import Play.autoImport._
import PlayKeys._

```

**Comment by marcospereira** [comment]
Hey, @wjtan thanks for reporting this problem.

Do you want to submit a pull request to fix this issue?

**Comment by wjtan** [comment]
I have created a pull request for this issue: https://github.com/playframework/playframework/pull/6034

**Comment by marcospereira** [comment]
#6034 merged and backported to 2.5.x.

Thanks, @wjtan.
